aws/data_aws_unity_catalog_policy_test.go

```diff
@@ -0,0 +1,116 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/databricks/terraform-provider-databricks/qa"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDataAwsUnityCatalogPolicy(t *testing.T) {
+	d, err := qa.ResourceFixture{
+		Read:        true,
+		Resource:    DataAwsUnityCatalogPolicy(),
+		NonWritable: true,
+		ID:          ".",
+		HCL: `
+		aws_account_id = "123456789098"
+		bucket_name = "databricks-bucket"
+		role_name = "databricks-role"
+		kms_name = "databricks-kms"
+		`,
+	}.Apply(t)
+	assert.NoError(t, err)
+	j := d.Get("json")
+	assert.Lenf(t, j, 821, "Strange length for policy: %s", j)
```

Comment:
This is a weird assertion. If the policy changes, we should see the diff from expected. This will only show the new policy. I would only include the Equal assertion here and below.